docs/understanding-radicle/glossary.md

```diff
@@ -47,28 +40,27 @@ to disseminate information. Participants in the Radicle network share and spread
 data by keeping redundant copies of projects locally and sharing updates with
 peers. This allows data to be disseminated according to use and value: the more
 peers who are interested in certain data the more available it is made to the
-network. See [Gossip][hiw-rm].
+network.
 
 ## maintainer
 A [user][us] who manages and pushes updates to the canonical view of a project.
 A maintainer is explicitly associated with the [project][pr] at the time of
-creation or later, via the [project][pr] metadata (see [Data Model][hiw-dm]),
+creation or later, via the [project][pr] metadata ,
 but there may be multiple maintainers of a project, all maintaining their own
 trees. At creation, the creator of the [project][pr] is the sole maintainer.
 
 ## main
 How Radicle refers to the default development branch of a project.
 
 ## monorepo
```

Comment:
Under the new protocol, there won't be a monorepo per se. As in, there won't be a single "Git monorepo", but rather there will be the structure:
```
storage/         # root
  <project>/... # a git directory for a single project
  <project>/... # another git directory for another project
```

Semantically, the `storage` part can be thought of as a monorepo :sweat_smile: But maybe we get rid of that term. I began to stop using it and opted for "Radicle storage" to avoid confusion.